src/sdk/pynni/nni/analysis_utils/topology/torch/graph_from_trace.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
what is the difference between this file and `https://github.com/microsoft/nni/blob/master/src/sdk/pynni/nni/_graph_utils.py`?

Reply:
graph_from_trace has made some modifications to the network structure for the clarity of the visual graph, such as unpack tensor tuple handly.  graph_from_trace also can build the graph according to the existing Traced graph, so there is no conflict between the two modules, because we only trace the model once. In addition, _graph_utils build the graph by indexing the layers' name which is dangerous(https://github.com/microsoft/nni/issues/2485). I'm not sure if I explain this clearly. if not, let's discuss this in the meeting.